libvast/src/format/arrow.cpp

```diff
@@ -163,8 +163,9 @@ reader::read_impl(size_t max_events, size_t max_slice_size, consumer& f) {
     // batch, but it's a good enough heuristic to prevent users from running
     // into crashes. A proper fix for this requires casting to VAST-compatible
     // record batch.
-    if (read_result->batch->schema()->metadata()->FindKey("VAST:name:0")
-        == -1) {
+    const auto& metadata = *read_result->batch->schema()->metadata();
+    if (metadata.FindKey("Tenzir:name:0") == -1
```

Comment:
```suggestion
    if (metadata.FindKey("TENZIR:name:0") == -1
```

I find `TENZIR` or `tenzir` arguably easier to remember.

Reply:
I'd prefer either Tenzir or tenzir. Do you have a strong opinion on this before I merge this?

Reply:
Kind of. Arrow uses `ARROW` and I would expect that we follow the naming scheme. So I'd say the most natural one would actually be `TENZIR`. What's your rationale for `Tenzir`?

Reply:
I was referring to unit test output in this very PR.

Reply:
Right, the undocumented and internal `ARROW:extension:{name,metadata}` keys.

I don't have too strong of an opinion on this, and it seems like you have, so I'll just make the change.

Reply:
Thanks!

Reply:
Implemented in 2d9ccc0148857accec408e3b3f1cdaeba4e49346